### ISSUE TITLE
Choose the audio stream and the subtitle stream remotely.

### ISSUE THREAD

**Issue by Lumenol** [jellyfin/jellyfin-kodi#309]
**Describe the bug**

I cannot choose the audio stream or the subtitles using a remote interface. I can see that it is used but if I change it it is not taken into account by kodi.

**To Reproduce**

1. Activate the remote control of kodi.
2. Start a movie with multiple audio streams or subtitles.
3. Tried to change the audio or subtitle language from the remote interface.
4. Observe that change does not happen.

**Expected behavior**

When I change the audio stream or the subtitles from a remote interface, the change is done in kodi.

**Logs**

```
2020-05-30 18:29:02.112 T:16656  NOTICE: JELLYFIN.player -> INFO::jellyfin_kodi\player.py:161 Setting audio: 3 subs: None
2020-05-30 18:29:02.117 T:16656   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: unsupported operand type(s) for -: 'unicode' and 'int'
                                            Traceback (most recent call last):
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\monitor.py", line 261, in onNotification
                                                self.general_commands(data)
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\monitor.py", line 388, in general_commands
                                                self.player.set_audio_subs(args['Index'])
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\player.py", line 170, in set_audio_subs
                                                self.setAudioStream(audio - 1)
                                            TypeError: unsupported operand type(s) for -: 'unicode' and 'int'
                                            -->End of Python script error report<--
2020-05-30 18:29:04.413 T:16656  NOTICE: JELLYFIN.player -> INFO::jellyfin_kodi\player.py:161 Setting audio: None subs: 6
2020-05-30 18:29:04.413 T:16656   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: unsupported operand type(s) for -: 'unicode' and 'int'
                                            Traceback (most recent call last):
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\monitor.py", line 261, in onNotification
                                                self.general_commands(data)
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\monitor.py", line 390, in general_commands
                                                self.player.set_audio_subs(None, args['Index'])
                                              File "C:\Users\crede\AppData\Roaming\Kodi\addons\plugin.video.jellyfin\jellyfin_kodi\player.py", line 189, in set_audio_subs
                                                self.setSubtitleStream(subtitle - tracks - 1)
                                            TypeError: unsupported operand type(s) for -: 'unicode' and 'int'
                                            -->End of Python script error report<--
2020-05-30 18:29:10.523 T:16656  NOTICE: JELLYFIN.monitor -> INFO::jellyfin_kodi\monitor.py:367 [ command/PlayPause ]
2020-05-30 18:29:11.071 T:16248 WARNING: CRenderManager::WaitForBuffer - timeout waiting for buffer
2020-05-30 18:29:14.527 T:14732 WARNING: Previous line repeats 6 times.
2020-05-30 18:29:14.527 T:14732  NOTICE: VideoPlayer: OnLostDisplay received
2020-05-30 18:29:14.530 T:14732  NOTICE: VideoPlayer: OnResetDisplay received
2020-05-30 18:29:15.041 T:16248 WARNING: CRenderManager::WaitForBuffer - timeout waiting for buffer
2020-05-30 18:29:15.838 T:14732 WARNING: Previous line repeats 1 times.
```

**Screenshots**

**System (please complete the following information):**
 - OS: [e.g. Android, Debian, Windows]
 - Jellyfin Version: 10.5.5
 - Kodi Version: 18.7
 - Addon Version: 0.5.6

**Additional context**

**Comment by Lumenol** [comment]
I found a solution by correcting 'set_audio_subs' in player.py.
However the poster in the remote interface does not update the first time when I activate or deactivate the subtitles. You have to do the manipulation again.
Similarly if I make the changes in kodi the display of the activation and deactivation of the subtitles is not done in the remote interface.

**Comment by mcarlton00** [comment]
Interesting.  I wonder if it's just an event that we're not capturing/forwarding to the server properly.  If you'd like, feel free to open a PR as a draft with your fix and I can help take a look at it.  Or we can just merge your first fix and treat that as a separate issue.

**Comment by Lumenol** [comment]
After doing some additional testing. I noticed that the modifications in Kodi only went back under certain conditions. I added the detail in the PR.

**Comment by mcarlton00** [comment]
Fixed by #310
